### DISCUSSION TITLE
Cleaning up a server: /root/.config/onedrive

### DISCUSSION THREAD

**Discussion by rrodrigueznt** [abraunegg/onedrive#2272] [General]
Hi! Looking for cleaning up and understanding all concepts related to OneDrive functioning, I found an OneDrive config folder under /root as ```/root/.config/onedrive```.

To the best of my current understanding, it has no relation with the current OneDrive instance/instances running on that server **installed as a regular user**, and it's just a scrap of an old installation done a time ago and no longer used. Thus, it would be safe to remove it. I found a /OneDrive folder under /root with no contents and no OneDrive service is running for root user.

Please, should I check anything else before deleting it? Thanks!

**Comment by abraunegg** [comment]
@rrodrigueznt 
> Please, should I check anything else before deleting it?

It seems like you have already done all the right checks - checking for any systemd process running as 'root' and/or any active 'onedrive' process would be the 2 steps I would do before removing those two folders.
